app/_indices/konnect-platform.yaml

```
@@ -39,4 +39,5 @@ sections:
     items: 
       - path: /konnect-platform/debugger/
       - path: /konnect-platform/debugger-spans/
```

```suggestion
      - path: /gateway/debugger/
      - path: /gateway/debugger-spans/
```
Though not sure if these should stay in this Konnect Platform index.